environment.yml

```diff
@@ -18,13 +18,15 @@ dependencies:
 - joblib
 - psutil
 - numexpr
-- traits>=4.6.0
-- pyface>=6
-- traitsui>=6
-- imageio>=2.6.1
+- traits
+- pyface>=6,<7
+- traitsui
+- imageio
+- numpydoc
```

Comment:
We can just get rid of this actually, but that can be another PR. This one seems to be headed toward green (other than our recent pip-pre 3.8 segfaults that are unrelated)

Reply:
What do you mean? Remove these four packages completely from the env?

Reply:
No just `numpydoc`

Reply:
OK, I'll remove `numpydoc`. What about the other packages I moved from `pip` to `conda`? OK or revert?